pkg/machine/wsl/usermodenet.go

```diff
@@ -353,13 +353,27 @@ func changeDistUserModeNetworking(dist string, user string, image string, enable
 		return err
 	}
 
+	if !podmanSetup {
+		if err := appendSystemdConfig(dist); err != nil {
```

Comment:
you put this in the same function which calls `writeWslConf`. However, it’s unexpected to me to see these changes in a function named `changeDistUserModeNetworking` . Maybe this `appendSystemdConfig` can be moved to `SetProviderAttrs`, or somewhere in `CreateVM`? This is not something we’ll change after the VM is created.

Reply:
:+1: moved